source/Tasks/CompileDatumRsop.build.ps1

```diff
@@ -31,41 +31,60 @@ param
     [string]
     $RsopFolder = (property RsopFolder 'RSOP'),
 
+    [Parameter()]
+    [string]
+    $RsopWithSourceFolder = (property RsopFolderWithSource 'RsopWithSourceFolder'),
+
     [Parameter()]
     [string]
     $ModuleVersion = (property ModuleVersion '')
 )
 
 task CompileDatumRsop {
+
+    Clear-DatumCache #otherwise this task will not generate new RSOP data
```

Comment:
Note for me: find out why. If we have the cache, we should have the RSOP and could avoid another build of RSOP.